jetty-core/jetty-deploy/src/main/config/etc/jetty-core-deploy.xml

```diff
@@ -33,7 +33,7 @@
             </Call>
           </Set>
           <Set name="deferInitialScan" property="jetty.deploy.deferInitialScan"/>
-          <Set name="scanInterval" property="jetty.deploy.scanInterval"/>
+          <Set name="scanInterval"><Property name="jetty.deploy.scanInterval" default="1"/></Set>
```

Comment:
I think the file stays as it is, and we change the values in the `{$ee}-deploy.mod` files to be `0`, in accordance with the (new) policy of making jetty static deployment by default.